src/lit-html.ts

```diff
@@ -212,7 +237,21 @@ export class Template {
    * Returns a string of HTML used to create a <template> element.
    */
   private _getHtml(strings: TemplateStringsArray, svg?: boolean): string {
-    const html = strings.join(exprMarker);
+    const l = strings.length;
+    const a = new Array((l * 2) - 1);
```

Comment:
Did you mean to create a pre-sized Array here? The first time you push onto this Array, it will insert at `(l * 2) - 1`. `join` working later on is kind of a happy accident.

Reply:
Damn, my misunderstanding of the Array constructor. My intention was to pre-allocate the array to the right size.

Reply:
fixed